src/psyclone/psyir/nodes/omp_directives.py

```diff
@@ -705,7 +705,7 @@ def _get_private_clauses(self):
                     if loop_ancestor:
                         # The assignment to the variable is inside a loop, so
                         # declare it to be private
```

Comment:
Please extend the comment. Something like "...to be private. Ensure we only use the root variable name in case this is an access of a structure member (see 3.2.1 of the OMP spec. v.5.2)."

Reply:
Added